### PR TITLE
Ato message

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR adds steps to enable a login banner from the system console.

- [Palette](https://deploy-preview-3738--docs-spectrocloud.netlify.app/enterprise-version/system-management/login-banner/)
- [VerteX](https://deploy-preview-3738--docs-spectrocloud.netlify.app/vertex/system-management/login-banner/)
- [Updated Warning Box Message](https://deploy-preview-3738--docs-spectrocloud.netlify.app/tenant-settings/login-banner/)

## Changed Pages

💻 [Preview URL for Page](https://deploy-preview-3738--docs-spectrocloud.netlify.app/enterprise-version/system-management/login-banner/)

## Jira Tickets

🎫 [DOC-1356](https://spectrocloud.atlassian.net/jira/software/c/projects/DOC/boards/62?selectedIssue=DOC-1356)

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-1356]: https://spectrocloud.atlassian.net/browse/DOC-1356?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: _partials/self-hosted/_login-banner.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+partial_category: self-hosted
+partial_name: login-banner
+---
+
+
+
+
+{props.edition} allows you to add a Login banner, also known as an Authority to Operate (ATO) banner or System Message, to the login page. This banner can display important information to users before they log in to the system console or an individual tenant.
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply _partials/self-hosted/_login-banner.mdx:9]: Oh interesting. It didn't catch the next line containing the (ATO)

File: _partials/self-hosted/_login-banner.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+## Prerequisites
+
+- Access to the Palette system console.
+<!-- prettier-ignore -->
+- System administrator permissions, either a Root Administrator or Operations Administrator. Refer to the 
+  <PaletteVertexUrlMapper
+     edition={props.edition}
+     text="System Administrators"
+     url="/system-management/account-management"
+   /> guide. page to learn more about system administrator roles.
```
Comment (caroldelwing) [inline _partials/self-hosted/_login-banner.mdx:30]:
```suggestion
   /> page to learn more about system administrator roles.
```

File: _partials/self-hosted/_login-banner.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+
+
+
+{props.edition} allows you to add a Login banner, also known as an Authority to Operate (ATO) banner or System Message, to the login page. This banner can display important information to users before they log in to the system console or an individual tenant.
+
+
+:::warning
+
+Setting a login banner at the system console will override any tenant-specific login banners configured. Refer to the <VersionedLink text="Tenant Login Banner" edition={props.edition}  url="/tenant-settings/login-banner/"/> guide to learn more about tenant-specific login banners.
```
Comment (caroldelwing) [inline _partials/self-hosted/_login-banner.mdx:14]: Does it make sense to use edition={props.edition} inside the versioned link? :B

File: _partials/self-hosted/_login-banner.mdx (hunk 1 above)
Comment (caroldelwing) [inline _partials/self-hosted/_login-banner.mdx:9]:
```suggestion
{props.edition} allows you to add a login banner, also known as an Authority to Operate (ATO) banner or system message, to the login page. This banner can display important information to users before they log in to the system console or an individual tenant.
```

File: _partials/self-hosted/_login-banner.mdx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+:::
+
+
+Use the following steps to add a login banner to the {props.edition} login page.
+
+## Prerequisites
+
+- Access to the Palette system console.
```
Comment (caroldelwing) [inline _partials/self-hosted/_login-banner.mdx:23]: The link is missing (I think). Maybe we could use the component here as well to say "Access to the Vertex system console"  :)

File: _partials/self-hosted/_login-banner.mdx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+<!-- prettier-ignore -->
+1. Log in to the {props.edition} system console. Refer to the
+    <PaletteVertexUrlMapper
+        edition={props.edition}
+        text="Access System Console"
+        url="/system-management/#access-the-system-console"
+      /> guide for more information.
+
+2. From the left **Main Menu** select **Administration**.
```
Comment (caroldelwing) [inline _partials/self-hosted/_login-banner.mdx:46]:
```suggestion
2. From the left **Main Menu**, select **Administration**.
```

**Review by caroldelwing — APPROVED** [review]
Just tiny suggestions :rocket:

File: _partials/self-hosted/_login-banner.mdx (hunk 3 above)
Comment (karl-cardenas-coding) [reply _partials/self-hosted/_login-banner.mdx:14]: Yup, not needed.

**Pull request merged**